docs/guide/building-a-basic-scene.md

```diff
@@ -4,38 +4,54 @@ type: guide
 layout: docs
 parent_section: guide
 order: 3
-show_guide: true
 ---
 
-Let's first start building a scene using [primitives][primitives], the basic building blocks of A-Frame with familiar HTML syntax. Under the hood, primitives are aliases [entities][entity] that proxy HTML attribute values to component property values. A-Frame is bundled with a handful of primitives for common use cases such as backgrounds, images, meshes, models, images, and videos.
+Let's first start building a scene using the basic [primitive][primitives]
+building blocks. Primitives are the appetizer of A-Frame that ease newcomers
+into the underlying entity-component-system.
+
+<!--toc-->
 
 ## Adding a Box
 
-The simplest scene would be a scene containing a box primitive:
+A *Hello World* scene is a scene with a box:
 
 ```html
 <a-scene>
   <a-box color="#6173F4" width="4" height="10" depth="2"></a-box>
 </a-scene>
 ```
 
-Just like with regular HTML elements, each attribute of the entity maps to one value. We can define a color, width, height, and depth of `<a-box>`. To see more attributes that `<a-box>` and other geometric primitives can accept, check out the [common mesh attributes][mesh].
+Just like with regular HTML elements, we can configure the element by setting
+HTML attributes. Here, we define the color, width, height, depth of `<a-box>`.
+See the [`<a-box>`][box] documentation to see the available attributes.
 
-Once we open up our scene, the default control setup allows us to look and walk around. To look around, we can drag the mouse or just look around with a mobile device or a Rift. To walk around, we can use the WASD keys. Then to [enter VR][mozvr], click on the Enter VR button.
+For flat displays, the default control scheme lets us to look around by
+click-dragging the mouse and move around by using the `WASD` keys. Upon
```

Comment:
Extra "to" on line 29.

Suggestion: "For flat displays, the default control scheme lets us look around by click-dragging the mouse and move with the `WASD` keys".